string name
time r_time
int8 state
string In_topic
string Out_topic
duration dead_line
